configs/sites/tier1/noaa-aws/packages.yaml

```diff
@@ -1,115 +1,114 @@
 packages:
-### Modifications of common packages
-  # Pin flex to avoid duplicate packages
-  flex:
-    version: ['2.6.4']
-### All other external packages listed alphabetically
   bash:
     externals:
-    - spec: bash@4.2.46
+    - spec: bash@4.4.20
       prefix: /usr
   berkeley-db:
     externals:
-    - spec: berkeley-db@5.3.21
+    - spec: berkeley-db@5.3.28
       prefix: /usr
   cpio:
     externals:
-    - spec: cpio@2.11
+    - spec: cpio@2.12
       prefix: /usr
   diffutils:
     externals:
-    - spec: diffutils@3.3
+    - spec: diffutils@3.6
       prefix: /usr
   ecflow::
```

Comment:
no external ecflow - but need external qt@5

Reply:
That is now in common/modules.yaml and common/packages.yaml:
```
    qt:
      require: '@5'
```

Reply:
yes but you should configure the external qt5 package so that you don't need to build it (often doesn't work as expected). since you were able to build ecflow outside of spack, the necessary qt 5 libraries must be somewhere on the system

Reply:
OK, I'm not sure how it worked. I intentionally didn't install:
```
#yum install -y qt5-qtbase-devel
#yum install -y qt5-qtsvg-devel
#yum install -y xorg-x11-xauth
#yum install -y xorg-x11-apps
#yum install -y perl-IPC-Cmd
#yum install -y gettext-devel
#yum install -y m4
```
Which we were supposed to install each time we start cloud instance.
Ecflow was installed using qt5 (which will be lost each time we logout).
Are you saying that we have to install qt5 again each time we login (like before)?
ecflow is still there, on permanent place (/contrib/spack-stack-rocky8/ecflow-5.8.4/)

As for m4, gettext, ... they were successfully installed by spack-satck-1.8.0.

Reply:
I think you need to install those packages every time, but you can easily create a bootstrap script and run that in your parallelworks config

Reply:
That's OK, but why do we need to install them anyway, spack stack does not depend on them?

Reply:
qt5 does depend on them, and you need qt5 to build ecflow+ui - you can choose to let spack build all of that, not sure it works though